packages/client/src/types.ts

```diff
@@ -36,6 +36,15 @@ export interface WorkflowExecutionInfo {
   taskQueue: string;
   status: { code: proto.temporal.api.enums.v1.WorkflowExecutionStatus; name: WorkflowExecutionStatusName };
   historyLength: number;
+  /**
+￼   * Size of Workflow history in bytes until the current Workflow Task.
```

Comment:
`until the current Workflow Task` don't really make sense in this structure. There is no "current workflow task". Similar for `This value changes during the lifetime of an Execution`. I don't think it adds real value in this context.

Last thing: `historySize` is available in server versions >= 1.20 (ie. starting with 1.20, not greater than)

Reply:
Updated comment